charts/prow/temp/marque_deployment.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
I recently deleted marque in favor of kube-lego, which already has a helm chart. Is it easy to include one helm chart in another?